pdoc/html_helpers.py

```diff
@@ -323,7 +326,8 @@ def admonitions(text, module, limit_types=None):
         See: https://python-markdown.github.io/extensions/admonition/
         """
         substitute = partial(re.compile(r'^(?P<indent> *)\.\. ?(\w+)::(?: *(.*))?'
-                                        r'((?:\n(?:(?P=indent) +.*| *$))*)', re.MULTILINE).sub,
+                                        r'((?:\n(?:(?P=indent) +.*| *$))*[^\n|\r|\r\n])*',
```

Comment:
`[]` itself denotes a disjunctive list of characters, so `[^\r\n]` suffices.
```suggestion
                                        r'((?:\n(?:(?P=indent) +.*| *$))*[^\r\n])*',
```

Reply:
Told ya I wasn't a regex expert ;)